README.md

```diff
@@ -2,6 +2,12 @@
 
 An interactive tour of the Gleam programming language.
 
+## Build
+
+### Prerequisites
+
+- `node`
```

Comment:
Remove these headings please. Perhaps a sentence saying "A JavaScript runtime is required to run this program. NodeJS is used by default."?

Reply:
https://github.com/gleam-lang/language-tour/pull/152/commits/d8a35acbccaf4dbc2aa6e4da2cc27b486e678002